patches/react-native-fast-image+8.1.5.patch

```diff
@@ -0,0 +1,174 @@
+diff --git a/node_modules/react-native-fast-image/android/src/main/java/com/dylanvann/fastimage/FastImageViewModule.java b/node_modules/react-native-fast-image/android/src/main/java/com/dylanvann/fastimage/FastImageViewModule.java
```

Comment:
Too many code on a patch and the PR is from early 2019.
IMO we should do a fork.

Reply:
Agree